code/api/src/modules/subscription/mutations.js

```diff
@@ -27,4 +26,24 @@ export const subscriptionRemove = {
     }
   },
   resolve: remove
-}
\ No newline at end of file
+}
+
+// nextOrder Update
+export const nextOrderUpdate = {
+  type: SubscriptionType,
+  args: {
```

Comment:
so we need to pass this a nextOrder, a user, and a crate to update a nextOrder

Reply:
exactly! 😄